plugins/jenkins-backend/src/service/jenkinsApi.ts

```diff
@@ -165,7 +168,10 @@ export class JenkinsApiImpl {
 
     return {
       ...project,
-      lastBuild: this.augmentBuild(project.lastBuild, jobScmInfo),
+      lastBuild:
+        project.lastBuild === null
```

Comment:
Would like something like

```ts
project.lastBuild
  ? this.augmentBuild(project.lastBuild, jobScmInfo)
  : undefined;
```

Preferring to treat "as a boolean", and avoiding null :)

Reply:
I can set the  `project.lastBuild?....`, but I cannot set "undefined" as jenkins send null and the same object is used as base for both the jenkins mapping and backstage object. 
Just did the change.